lib/season_statistics.rb

```diff
@@ -9,14 +9,125 @@ def biggest_bust(season)
       ps_game = games.find_all do |game|
         @games[game.game_id].type == "Postseason"
       end
-    require'pry';binding.pry
     end
   end
-end
 
-    # post_season_games = games_by_season[season].find_all do |game|
-    #   game.season == "Postseason"
-    # end
-    # regular_season_games = games_by_season[season].find_all do |game|
-    #   game.season == "Regular Season"
-    # end
+  def most_tackles(season_id)
+    filtered_games = @game_teams.find_all do |game|
+      @games[game.game_id].season == season_id
+    end
```

Comment:
We could pull this out to a helper since we use it in every method in this module:
```
filtered_games = @game_teams.find_all do |game|
   @games[game.game_id].season == season_id
end
```